Some packages in requirements require additional SO libraries to be installed

First time I tried to run the `pip install -r requirements.txt` I've encountered some problems, the first one been `pylibmc`:

```
gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -DUSE_ZLIB -I/usr/include/python2.7 -c src/_pylibmcmodule.c -o build/temp.linux-x86_64-2.7/src/_pylibmcmodule.o -fno-strict-aliasing

In file included from src/_pylibmcmodule.c:34:0:

src/_pylibmcmodule.h:42:36: fatal error: libmemcached/memcached.h: No such file or directory

compilation terminated.

error: command 'gcc' failed with exit status 
```

OBS: I'll check back what really are the ones that have this additional libraries and update the issue them, but for now this is the one that hit me with an error

--- a/README.markdown
+++ b/README.markdown
@@ -45,7 +45,7 @@ Tools to consider
 Installation
 ------------
 
-First you should install some OS libraries required for some packages, this can vary on each OS, but if you're on Ubuntu, then this should do the trick for you:
+First you should install some OS libraries required for some packages, this can vary with each OS, but if you're on Ubuntu 14.04, then this should do the trick for you:
 
     sudo apt-get install python-dev libmemcached-dev zlib1g-dev libpq-dev
 

